<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>962</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>941</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>登录区域</string>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>24</y>
       <width>931</width>
       <height>73</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="5">
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>真实</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLineEdit" name="lineEdit_pwd"/>
      </item>
      <item row="1" column="1" colspan="5">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Submit</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>PassWord:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Sumbit Name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="lineEdit_name"/>
      </item>
      <item row="0" column="0" rowspan="2">
       <widget class="QGraphicsView" name="graphicsView"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>651</width>
      <height>321</height>
     </rect>
    </property>
    <property name="title">
     <string>表格显示</string>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>631</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>941</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>结果显示</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>911</width>
       <height>81</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QLabel" name="dq_qishu">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>1063590</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_22">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>投注金额：</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="dq_jine">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>01234567</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>投注基数：</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="dq_jishu">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>10</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="label_7">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>连续错误：</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QLabel" name="lxcw">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>7</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lable11">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>当前收益：</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="shouyi">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>1000000000</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_24">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>投注模式：</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="dq_moshi">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>真实</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="label_20">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>投注大小：</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLabel" name="dq_daxiao">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>大</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QLabel" name="label_19">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>刷新倒计时：</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <widget class="QLCDNumber" name="lcdNumber">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="digitCount">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>当前期数：</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>941</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>上期结果显示</string>
    </property>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>911</width>
       <height>81</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0">
       <widget class="QLabel" name="label_16">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>上期结果：</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_12">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>投注基数：</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="sq_jishu">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>10</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="sq_jine">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>01234567</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="sq_qishu">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>1063590</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_26">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>投注金额：</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="sq_jieguo">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>1000000000</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_28">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>投注大小：</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="sq_daxiao">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>大</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="label_30">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>投注模式：</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLabel" name="sq_moshi">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>真实</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_18">
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>上期期数：</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>380</y>
      <width>281</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>动态信息</string>
    </property>
    <widget class="QListView" name="listView">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>241</width>
       <height>281</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
